string_lab.js

```diff
@@ -0,0 +1,155 @@
+// 1. `console.log` the first character of a string.
+
+let x = "This is my string assignment";
+console.log(x[0]);
+
+// 2. `console.log` the length of a string.
+
+console.log(x.length);
+
+// 3. `console.log` the last character of any string.
+
+console.log(x[x.length -1]);
+
+// 4. Create a code block that takes a single string variable, and returns a copy of the string with the last letter capitalized. For example:
+//
+// ```js
+// 'jimmy'
+// // => 'jimmY'
+// ```
+
+// let name1 = "raisin";
+// name2 = name1.slice(0,5) + name1[5].toUpperCase();
+// console.log(name2);
+
+// 5. Create a drEvil code block that will take a single number variable, and log the '<variableAmount> dollars',
+// your code block should add '(pinky)' at the end of the amount if it's 1 million. For example:
+//
+// ```js
+// let amount = 10
+// // => 10 dollars
+// let amount = 1000000
+// // =>  1000000 dollars (pinky)
+// ```
+
+for (let amount = 10; amount <= 1000000; amount *= 10){
+          if (amount === 1000000){
+        console.log(amount + " dollars (pinky)");
+      } else {
+        console.log(amount + " dollars");
+      }
+}
+
+// 6. Create a `verbing` code block. It should take a single string variable. If its length is at least 3, it should add 'ing' to its end, unless
+// it already ends in 'ing', in which case it should add 'ly' instead. If the string length is less than 3, it should leave it unchanged.
+// For example:
+//
+// ```js
+// verbing('box')
+// // => 'boxing'
+// verbing('train')
+// // => 'training'
+// verbing('swimming')
+// // =>  'swimmingly'
+// verbing('go')
+// // =>  'go'
+//
+let verbing = "go";
+console.log(verbing.length);
+
+for (let vb = 0; vb < 1; vb ++){
+    if (verbing.includes("ing")){
+    console.log(verbing + "ly");
+  }else if (verbing.length >= 3) {
+    console.log(verbing + "ing");
+  }else {
+    console.log(verbing);
+  }
+}
+
+// 7. Create a `withoutLast` code block that takes a single string variable, and returns a copy of the string without the last word.
+//
+let withoutLast = "The cat and the fiddle"
+withoutLast2= withoutLast.split(" ").slice(0, -1);
+console.log(withoutLast2);
```

Comment:
if you do withoutLast2.join(" ") => this will print out "The cat and the fiddle" as one string instead of an array of items.